recipes/pytango/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pytango" %}
+{% set version = "9.3.3" %}
+{% set sha256 = "c09fc7a44cac8c2d95d3ad39ba644998e8f468c730dc495443a74de05a07e9f9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cppzmq
+    - cpptango >=9.3,<9.4
+    - boost
+    - numpy
```

Comment:
The original pytango conda recipe was specifying numpy >=1.1
Shouldn't we keep that for the people who will be manually building the package and would like to install by mistake pytango on an environment where a numpy < 1.1 was installed?

Reply:
I was only thinking about conda-forge when updating the recipe. I know that numpy is pinned to 1.16.
Not sure who could build a package with a version older than 1.1 (it's from 2008). But it doesn't cost much to leave it and I guess it gives some useful information. I'll add it back.